plane/test_fit_plane.py

```diff
@@ -43,6 +43,49 @@ def test_fit_mapping(self):
                 self.assertAlmostEqual(v[i], fp.v[i])
                 self.assertAlmostEqual(h[i], fp.h[i])
 
+    def test_uv_check(self):
+        # 88 degree angles
+        u = [1,0,0]
+        v = [np.cos(np.deg2rad(88)),np.sin(np.deg2rad(88)), 0]
+        h = [10,15,12]
+        xyz = [np.array(u) * p[0] + np.array(v) * p[1] + np.array(h) for p in self.template_center_positions_uv_pix]
```

Comment:
This test can be simplified

fp = FitPlane(u_mm = [1,0,0], v_mm = [..], h_mm = [0,0,0])

And simplify the checks

fp.get_xyz_from_uv([1,1])

Don't need to add an enumerate